CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+
+- Rename instrumentation examples
+  ([#3206](https://github.com/open-telemetry/opentelemetry-python/pull/3206))
```

Comment:
```suggestion

```
Why CHANGELOG for this?

Reply:
Well, it is a change in the examples that users read, since it is user-facing I think it should have a CHANGELOG entry. :shrug: 

Not completely sure, though, I can remove it if you think it is not necessary.

Reply:
How does this help them? I don't think this is needed.

Reply:
All right, removing it then :v: